CHANGELOG.md

```diff
@@ -1,8 +1,44 @@
 # Changelog
 
+## [v2.0.0-BETA-5](https://github.com/stargate/stargate/tree/v2.0.0-BETA-5) (2022-10-18)
```

Comment:
do you need to do something to the changelog? even if to just put a note that beta-5 and rc-1 are identical?

Reply:
No I am redoing the rc-1, will have another update in another PR :)